firebase-crashlytics-ndk/CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # Unreleased
+
```

Comment:
why there were no messages here?

Reply:
They already updated their changelog to display 19.4.3, the release had to move these back to unreleased, so, there's no change here.